src/modules/ui/bottom-block/bottom-block.view.ts

```diff
@@ -114,6 +123,27 @@ class BottomBlockView extends View<IBottomBlockViewStyles>
     );
   }
 
+  addControl(key: string, element: HTMLElement, position?: 'left' | 'right') {
+    this._additionalButtons[key] = element;
```

Comment:
Where you use this?

Reply:
This is in case we need getter for additional buttons. How do you thing, should I implement it or just remove this object?